src/Commands/AdminCommands/SendtochannelCommand.php

```diff
@@ -1,5 +1,5 @@
 <?php
-/**
```

Comment:
Copy / paste?
@MBoretto Check this class again and make the styling "pretty" :wink: